<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contacts</class>
 <widget class="QMainWindow" name="Contacts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contacts</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>unnamed.png</normaloff>unnamed.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="mode">
      <property name="font">
       <font>
        <underline>false</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="title">
       <string>Mode ::</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="Light_mode">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Light Mode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="Dark_mode">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>0</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Dark Mode</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>612</width>
         <height>304</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_1">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>full name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ADD">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DELETE">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DELETE_ALL">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>DELETE ALL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDark_Mood">
   <property name="icon">
    <iconset>
     <normaloff>moon.png</normaloff>moon.png</iconset>
   </property>
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
